src/shared/types/index.ts

```diff
@@ -43,7 +43,6 @@ interface LedgerCorrected extends LedgerXRPL {
     (Transaction | PseudoTransaction) & {
       metaData?: TransactionMetadata
       hash: string
-      date: number
```

Comment:
Why was this removed?

Reply:
This was added by mistake as I thought there's a date field in the expanded ledger response but actually there isn't and we can use the `close_time` instead